src/web/core/src/main/java/org/geoserver/web/wicket/DatePicker.js

```diff
@@ -0,0 +1,116 @@
+// below some variable and functions to map date format tokens from java to moment js eg. d -> D for day of month.
```

Comment:
This is something you added ? Should have the GeoServer license header then.

Reply:
fixed